server/journey/journeyModel.js

```diff
@@ -14,6 +14,10 @@ var JourneySchema = new mongoose.Schema({
   },
   hash: {
     type: String
+  },
+  userID: {
+    type: String,
+    required: true
```

Comment:
-We discussed having other fields here in the journey's DB 

<img width="344" alt="screen shot 2016-04-22 at 3 03 45 pm" src="https://cloud.githubusercontent.com/assets/14899517/14756040/7e431584-089b-11e6-812b-009ce406e980.png">